### PR TITLE
Update image sharpen using ghost opencv ws

### PR DESCRIPTION
**Pull request by dkdkwizard**
As title.
This branch should be used with new ghost.

### CODE REVIEW

File: src/web/app/lang/en.ts [hunk 1]
```diff
@@ -603,12 +603,14 @@ export default {
...
       sharpen: 'Sharpen',
       sharpness: 'Sharpness',
+      radius: 'Radius',
       crop: 'Crop',
       curve: 'Curve',
       start: 'Start',
       processing: 'Processing',
       invert: 'Invert Color',
       okay: 'Okay',
+      compare: 'Comparar',
```
Comment (net5566) [inline src/web/app/lang/en.ts:613]: typo

Comment (dkdkwizard) [reply src/web/app/lang/en.ts]: I mixed up en with es.

File: src/web/app/lang/zh-cn.ts [hunk 2]
```diff
@@ -603,12 +603,14 @@ export default {
       next: '下一步',
       sharpen: '锐化',
       sharpness: '鋭化强度',
+      radius: 'Radius',
```
Comment (ChenYanting) [inline src/web/app/lang/zh-cn.ts:606]: no translation?

File: src/web/app/views/beambox/Photo-Edit-Panel.tsx [hunk 3]
```diff
@@ -10,643 +9,549 @@ import Modal from 'app/widgets/Modal';
...
-            }
-            
-        } catch (err) {
-            console.log(err)
-            Progress.popById('photo-edit-processing');
-        }
+interface IProps {
+  element: HTMLElement,
+  src: string,
+  mode: 'sharpen' | 'crop' | 'curve',
```
Comment (ChenYanting) [inline src/web/app/views/beambox/Photo-Edit-Panel.tsx:38]: duplicate definition, could we define it somewhere making it reused?
https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types

Comment (dkdkwizard) [reply src/web/app/views/beambox/Photo-Edit-Panel.tsx]: I will add a literal type and export it.

**Review by ChenYanting — APPROVED** [review]
LGTM

**Pull request merged**